### PR TITLE
Health check CS

### PR DESCRIPTION
**Pull request by tsachzim**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by altmannyarden** [comment]
Notes from our review today:
1. playbook name, incident type name, pack name, script names
2. General Analysis tab in the layout - rename.
3. Move the Server configuration to Server overview tab
4. Integration tab - rename tab to pack something, change the pack id to pack name, create another section for enabled instances
5. Add file validation to health check log analysis

**Comment by yaakovi** [comment]
what is the difference between `HealthCheckInstalledPacks` and `HealthCheckPacksInstalled`?

File: Packs/HealthCheck/IncidentFields/incidentfield-Enabled_Integrations_Instances.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "defaultRows": [
+        {}
+    ],
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_enabledintegrationsinstances",
+    "isReadOnly": false,
+    "locked": false,
+    "name": "Enabled Integrations Instances",
```
[the remark this answers is not in the document]
Comment (tsachzim) [reply Packs/HealthCheck/IncidentFields/incidentfield-Enabled_Integrations_Instances.json:33]: Removed, good catch

**Review by yaakovi — APPROVED** [review]
approving, not editing all scripts.
when moving to XSOAR support, need to revisit all scripts implementations and outputs.

**Comment by altmannyarden** [comment]
> Notes from our review today:
> 
> 1. playbook name, incident type name, pack name, script names
> 2. General Analysis tab in the layout - rename.
> 3. Move the Server configuration to Server overview tab
> 4. Integration tab - rename tab to pack something, change the pack id to pack name, create another section for enabled instances
> 5. Add file validation to health check log analysis

Why is the pack id is "healthcheck"?? we said to change this name multiple times. We wish to create a pack name "Health Check" in the future that will contain both packs and future health check packs.

**Pull request merged**